themes/default/content/blog/scaling-apps-across-multiple-regions/index.md

```diff
@@ -0,0 +1,202 @@
+
+You can create your stacks by running:
+
+```bash
+pulumi stack init Pulumi.prodwu2.yaml
+pulumi stack init Pulumi.prodwe.yaml
+pulumi stack init Pulumi.stagungsea.yaml
+```
+
+Update your Pulumi configuration YAML files with their corresponding values.
+
+**Pulumi.prodwu2.yaml:**
+
+```yaml
+config:
+  appServiceTier: PremiumV2
+  appServiceSize: P1V2
+  environment: production
+  region: westus2
+```
+
+**Pulumi.prodwe.yaml:**
+
+```yaml
+config:
+  appServiceTier: PremiumV3
+  appServiceSize: P1V3
+  environment: production
+  region: westeurope
+```
+
+**Pulumi.stagingsea.yaml:**
+
+```yaml
+config:
+  appServiceTier: Standard
+  appServiceSize: S1
+  environment: staging
+  region: southeastasia
+```
+
+For configuration values that are shared across stacks, you should use ESC so that you can create them once and reference them everywhere. With ESC, your stack will look similar to this:
+
+```yaml
+config:
+  appServiceTier: PremiumV2
+  appServiceSize: P1V2
+  environment: production
+  region: westus2
+
+environment:
+  imports:
+  - tutorial-app-config
+```
+
+Here, `tutorial-app-config` has more configurations and you can add references to them in your stack without copying and pasting multiple lines.
+
+### Update Pulumi Code
+
+Update your Pulumi code to use the configurations from the Pulumi stack:
+
+```typescript
+import * as pulumi from "@pulumi/pulumi";
+import * as azure from "@pulumi/azure";
+
+// Define the configuration
+const config = new pulumi.Config();
+const appServiceTier = config.require("appServiceTier");
+const appServiceSize = config.require("appServiceSize");
+const environment = config.require("environment");
+const region = config.require("region");
+
+const planName = `my-app-service-plan-${environment}`;
+const resourceGroupName = `my-resource-group-${environment}`;
+
+// Create the resource group
+const resourceGroup = new azure.core.ResourceGroup(resourceGroupName, {
+    location: region,
+});
+
+// Create the App Service Plan
+const appServicePlan = new azure.appservice.Plan(planName, {
+    resourceGroupName: resourceGroup.name,
+    kind: "Linux",
+    reserved: true,
+    sku: {
+        tier: appServiceTier,
+        size: appServiceSize,
+    },
+});
+```
+
+### Deploy Stacks
+
+Deploy each stack to its respective region
+
+```bash
+pulumi up --stack prodwu2
+pulumi up --stack prodwe
+pulumi up --stack stagingsea
+```
+
+Now, you have seen how to use stacks to represent regions and environments. With this approach, the dynamism happens through your configuration and your code doesn’t have to change per region.
+
+To see a full application with more detail, click here.
+
+## Implementing High Availability
```

Comment:
Let's change this to "Considerations for High Availability" or similar, and reword the following paragraph just a bit. This section is more about the different aspects that architects must consider, and less about actual implementation.